src/perf/lib/SecNetPerfMain.cpp

```diff
@@ -139,6 +139,11 @@ QuicMainStart(
         if (QUIC_FAILED(Status)) {
             CxPlatDataPathUninitialize(Datapath);
             Datapath = nullptr;
+            //
+            // Must explicitly set binding to null, as CxPlatSocketCreateUdp
+            // can set the Binding variable even in invalid cases.
```

Comment:
Well, it should clear it then. Can you fix that with a separate PR?

Reply:
Yes, I can do that in a separate PR.